src/components/leaderboard/leaderboardTabs/index.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import { Tabs, Pagination } from 'antd';
-import LeaderboardTab, { TabItem } from '../leaderboardTab';
+import LeaderboardTab from '../leaderboardTab';
+import { TabItem } from '../leaderboardSpace';
```

Comment:
📌 (out of scope for this PR): it seems like many components within `components/leaderboard/` use these types. What if we stored them in a types file in `components/leaderboard/types` and referenced that instead?

Reply:
Could be worth doing in this PR actually, but you can ticket separately if you want.

Reply:
I think it's just one type used everywhere, moved that to a new file